packages/ui-pattern-block/src/components/Captions.tsx

```diff
@@ -34,7 +34,7 @@ export const Captions = ({
     };
 
     return (
-        <div className={joinClassNames(['tw-flex tw-flex-col tw-gap-1'])}>
+        <div className={joinClassNames(['tw-flex tw-flex-col tw-gap-1 tw-w-full'])}>
```

Comment:
Why is this `tw-w-full` needed

Reply:
Otherwise this happens now. Probably some change around the scoping/fondue changes caused this.

![image](https://github.com/Frontify/guideline-blocks/assets/30796791/f9e65ddd-53e5-4e30-94e1-6210cc784848)

Reply:
Ok fine